public/stylesheets/landingPage.css

```diff
@@ -45,19 +45,19 @@ h1{
 }
 
```

Comment:
Is there a way to add images, instead of adding in CSS?

Reply:
didn't get your point